pipelines/utilities/terraform_setup.sh

```diff
@@ -24,4 +24,6 @@ if [[ "${TF_VAR_k8s_distro_name}" == "k3s" ]]; then
   terraform -chdir=test_framework/terraform/${LONGHORN_TEST_CLOUDPROVIDER}/${DISTRO} output -raw instance_mapping | jq 'map({(.name | split(".")[0]): .id}) | add' | jq -s add > /tmp/instance_mapping
 fi
 
+terraform -chdir=test_framework/terraform/${LONGHORN_TEST_CLOUDPROVIDER}/${DISTRO} output -raw controlplane_public_ip > /tmp/controlplane_public_ip
```

Comment:
Where `/tmp/controlplane_public_ip` will be used?

Reply:
For longhorn api client: https://github.com/longhorn/longhorn-tests/pull/1530/files#diff-c3e9bb86df59b862e75a6e5779a1f6631e809616c9de781e9a58d2b249d4dc06